sqlite_test.go

```diff
@@ -25,6 +25,8 @@ import (
 
 	"crawshaw.io/sqlite"
 	"crawshaw.io/sqlite/sqliteutil"
+
+	"github.com/pkg/errors"
```

Comment:
I'd rather not introduce this dependency in the tests. Could you just define a local error type that has a Cause method please?